thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/CompositePipelineConfigTranslator.java

```diff
@@ -379,16 +380,24 @@ private void fillInDetectorWrapperProperties(Map<String, Object> properties, Map
     return properties;
   }
 
+  //  Default schedule:
+  //  minute granularity: every 15 minutes, starts at 0 minute
+  //  hourly: every hour, starts at 0 minute
+  //  daily: every day, starts at 2 pm UTC
+  //  others: every day, start at 12 am UTC
   private String buildCron() {
+    // starts at random second to reduce task spike
```

Comment:
Can you include a comment on what task spike you are referring to here? and why is that a problem? probably mention how this correlates to high mysql updates.

Reply:
From database log all tasks are executed within the first three seconds which maps to the spike in mysql write spike since the task object is serialized in the database. We should distribute it.